content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/js.txt

```diff
@@ -5,4 +5,6 @@ touchui-limit-parsys.js
 touchui-configure-parsys-placeholder.js
 search-based-pathbrowser.js
 draggablelist.js
-dynamic-loader.js
\ No newline at end of file
+
```

Comment:
The whole file needs some rework I think because at least `touchui-limit-parsys.js` and `touchui-configure-parsys-placeholder.js` is no longer available in the `source` folder. I rather think the whole clientlib can be removed as this is incorrectly trying to aggregate stuff. Instead every aspect should be put in its own dedicated clientlib.